### DISCUSSION TITLE
New install, how do I get M117 to work, first layer notification?

### DISCUSSION THREAD

**Discussion by ForceConstant** [nlef/moonraker-telegram-bot#270] [General]
I see that M117 status message is supposed to work, but if I send command I do not get a notification. What am I missing?

Also is there a way to have bot send picture after first layer?

Thanks.

**Comment by aka13-404** [comment]
M117 does not send a message per default.
You are propably looking for this:
https://github.com/nlef/moonraker-telegram-bot/wiki/Interacting-with-klipper#send-custom-notifications-to-the-bot

**Reply by ForceConstant** [reply]
Thanks, I had seen that file, but don't really understand what it was saying, but realized I needed to enable '[respond]' section in my printer.cfg thanks.

**Comment by PhilBaz** [comment]
First layer notification. I've seen this asked a few places. 

Your slicer needs to insert a macro call at the correct location. Then simply make a macro with the message. In the example i show TB_FIRST is the macro name.

{if layer_num + 1==2} TB_FIRST {endif}


![Screenshot 2024-01-22 115932](https://github.com/nlef/moonraker-telegram-bot/assets/21558241/06183a35-3529-4337-a552-5a8585bc9a74)

**Comment by PhilBaz** [comment]
@nlef  The first layer message would make a great example for the documentation as it ties in the concept of macro calls from the slicer.
